components/PageControl/src/MDCPageControl.m

```diff
@@ -277,6 +277,11 @@ - (void)scrollViewDidScroll:(UIScrollView *)scrollView {
                    });
 
   } else if (scrolledPercentage >= 0 && scrolledPercentage <= 1) {
+
+    if (_numberOfPages == 0) {
```

Comment:
If this in a general guard statement, move to the beginning of the method?

If not, consider adding it in the if clause:
else if (scrolledPercentage >= 0 && scrolledPercentage <= 1 && _numberOfPages > 0)

A return in the middle of a method can easily get lost.

Reply:
Added it to the if clause above